src/client/Microsoft.Identity.Client/AppConfig/AuthorityInfo.cs

```diff
@@ -57,12 +61,32 @@ internal class AuthorityInfo
             }
         }
 
+        private AuthorityInfo(string host, string canonicalAuthority, AuthorityType authorityType, string userRealmUriPrefix, bool validateAuthority)
```

Comment:
Copy ctor since we don't want several authorities to point to the same AuthorityInfo (even though AuthorityInfo is now immutable).